travis/run_pylint.py

```diff
@@ -3,13 +3,14 @@
 
 from __future__ import print_function
 
-import os
-import sys
-
 import click
+import os
 import pylint.lint
+import sys
```

Comment:
Imports from the standard library should come before imports from third party packages.
See https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports